extensions/wikia/ContentReview/scripts/contentReviewTestMode.js

```diff
@@ -18,8 +19,13 @@ define(
 	}
 
 	function bindEvents() {
-		$('.content-review-test-mode-enable').on('click', enableTestMode);
-		$('#WikiaPage').on('click', '.content-review-test-mode-disable', disableTestMode);
+		if (win.skin === 'monobook') {
+			$('body').on('click', '.content-review-module-test-mode-enable', enableTestMode);
```

Comment:
To make sure we handle all entry points once they exist and they don't exist yet at the time the bind is done.